### PR TITLE
Add parse column count function

### PR DESCRIPTION
**Pull request by whmii**

### CODE REVIEW

**Comment by whmii** [comment]
Last PR in prep for https://github.com/thoughtbot/cask/pull/2

**Comment by whmii** [comment]
screen shot 2016-01-08 at 1 58 38 pm

File: app/assets/stylesheets/functions/_grid-parse-column-count.scss [hunk 1]
```diff
@@ -1,11 +1,13 @@
-// Parses the first argument of grid-column()
-
+// Parses the total number of columns from `$columns`
 @function grid-parse-column-count($span) {
+
```
Comment (kylefiedler) [inline app/assets/stylesheets/functions/_grid-parse-column-count.scss]: You don't seem do be adding lines in the above function, why here?

File: app/assets/stylesheets/functions/_grid-parse-column-count.scss [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   @if length($span) == 3 {
     $_total-columns: nth($span, 3);
     @return $_total-columns;
   } @else if length($span) == 2 {
     $_total-columns: nth($span, 2);
     @return $_total-columns;
+  } @else if length($span) >= 3 {
+    @error "`$column` had #{$span} values but should have no greater than 3";
```
Comment (kylefiedler) [inline app/assets/stylesheets/functions/_grid-parse-column-count.scss]: This error message is a bit confusing. I would want something telling me the right syntax.

**Comment by tysongach** [comment]
Look good, but before merging I think it would be valuable to respond to the comments above. I agree with the empty line formatting comment, Kyle made.

:+1:

**Pull request merged**